### DISCUSSION TITLE
Anime Search improvement

### DISCUSSION THREAD

**Discussion by FireBladeNooT** [pymedusa/Medusa#4368] [Ideas & Feature requests]
Hi,
Could you allow choosing or add using scene name for anime search?
Also it could be great to configure anime and choose if we want absolute number or not (S00E00).

For example:
Anime: My Hero Academia
Scene exception name: Boku no hero academia
Torrent looking for: Boku.no.Hero.Academia.S03E10.VOSTFR.720p.WEBRiP.x264-GODSPACE (mp4)

So in this example, the most efficient would be check "Anime" for nyaa search, not check "Absolute Numbering" and check "Scene Name for search".

To resume options:
- Anime (Remove absolute property)
- Absolute (New option to choose absolute number or normal)
- Scene Exception Name (New option to choose scene exception name or normal)
- All type of search (it is like actual search + search "scene name S00E00")

Or it could be options for each kind of search:
- Anime
- Normal numbering
- Absolute numbering
- Normal name
- Scene Exception Name

The main difficult in second proposal will be to managed all combinations possible and special cases like nothing checked.

What do you think about it?

Sincerely

**Comment by p0psicles** [comment]
Well all these options you wont need if your able to define your owm templated search strings per show and per scene exception title.

**Comment by FireBladeNooT** [comment]
What is the syntax proposed for the template search string?

**Comment by p0psicles** [comment]
Same as used for the post processing

**Comment by FireBladeNooT** [comment]
Ok so how made the difference between show name and scene exception names?

**Comment by p0psicles** [comment]
You will be able to add a template for each. I still need to create the ui components
